packages/react-pdf-node/src/index.js

```diff
@@ -1,15 +1,18 @@
 import fs from 'fs';
 import path from 'path';
-import { PDFRenderer, createElement, pdf } from '@react-pdf/core';
+import { createElement, pdf, PDFRenderer } from '@react-pdf/core';
 
 export default {
-  async render(element, filePath, callback) {
+  async renderToStream(element) {
     const container = createElement('ROOT');
     const node = PDFRenderer.createContainer(container);
 
     PDFRenderer.updateContainer(element, node, null);
 
-    const output = await pdf(container).toBuffer();
+    return await pdf(container).toBuffer();
+  },
+  async renderToFile(element, filePath, callback) {
+    const output = await this.renderStream(element);
```

Comment:
Should be `renderToStream` here right?

Reply:
Whoops, last minute change in a signature is never a good idea. Good catch!